src/main/java/plugins/nate/smp/SMPDatabase.java

```diff
@@ -0,0 +1,51 @@
+package plugins.nate.smp;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class SMPDatabase {
+    private Connection connection;
+
+    public void initialize() {
+        try {
+            String url = "jdbc:sqlite:" + new File(SMP.getPlugin().getDataFolder(), "smp.db").getAbsolutePath();
+            connection = DriverManager.getConnection(url);
+
+            try (Statement statement = connection.createStatement()) {
+                String sql = "CREATE TABLE IF NOT EXISTS PlayerSettings (" +
+                        "UUID TEXT PRIMARY KEY," +
+                        "Trusts TEXT," +
+                        "PvPEnabled INTEGER" +
+                        ");";
+
+                statement.execute(sql);
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void close() {
```

Comment:
I'd name this something like `closeSafely` because it's not just a direct reroute of the close call. I also think because you're checking connection.isClosed the exception will never be thrown, and that the `safely` name is a good one.
```suggestion
    public void closeSafely() {
```